ngraph/python/tests/coverage/compliance/helpers.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# The following tests contain expanded subgraphs representing an ONNX operator.
+ONNX_FUNCTION_TESTS = {
```

Comment:
These lists have to be updated after each onnx version change,
I wonder if we can force an user to update it.
What do you think about throw an exception if onnx != 1.9.0(current AFAIK) ?

Reply:
That sounds good, I'll give it a try.

Reply:
These tests aren't really testing the un-expanded operation. If our backedn doesn't support the operation, the tests will still potentially pass. This will produce a misleading entry in your report. I think we can avoid making this difficult to maintain list if we simply avoid mapping these tests to their original function op.

Reply:
Those are the tests for an expanded version of an ONNX function. Some of those models are already expanded, some of them are expanded in the ONNX importer. If I remove them from this list, I don't have a way to gather information about a single function/op being validated here.

Is there any other way to do it?